### DISCUSSION TITLE
Support nested dropdown

### DISCUSSION THREAD

**Discussion by NduatiK** [fluxonui/fluxon#14] [Feature Requests]
Navigation lists have an excellent API for nesting. Would it be possible to support something similar in dropdowns?

I am trying to implement something similar to Livebook's dropdown:
<img width="412" alt="image" src="https://github.com/user-attachments/assets/65cc07dc-35bf-4f0e-bc17-72890721981d" />

**Comment by NduatiK** [comment]
The following seems to work:
```html
<.dropdown
  label="Choose option"
  class="overflow-visible"
>
  <.dropdown_button>
    Option 1
  </.dropdown_button>

  <.dropdown
    open_on_hover
    hover_open_delay={100}
    hover_close_delay={300}
    placement="left-start"
    container_class="w-full"
  >
    <:toggle class="w-full">
      <.dropdown_button class="w-full">
        List
      </.dropdown_button>
    </:toggle>
    <.dropdown_button>Option 2.a</.dropdown_button>
    <.dropdown_button>Option 2.b</.dropdown_button>
  </.dropdown>
</.dropdown>
```

<img width="269" alt="image" src="https://github.com/user-attachments/assets/28eaefa0-94c2-4bca-9b13-292d9210fd5c" />

**Reply by andrielfn** [reply]
I plan to support nested dropdowns in the future, but I never meant to include them in this version. I'm honestly impressed that it worked 🤩 I haven't tested it yet, but I expect accessibility might not be great, and there could be bugs depending on how it's used.

Anyway, I do plan to "officially" support nested dropdowns at some point.

**Reply by NduatiK** [reply]
Yup, accessibility is poor. 

Thanks for looking into it.
